ppml/kms-client/DecryptFilesWithSpark-K8S-ALL.sh

```diff
@@ -0,0 +1,65 @@
+#set -x
+SPARK_DECRYPT_JAR_PATH=/ppml/trusted-big-data-ml/work/spark-3.1.2/examples/jars/spark-encrypt-io-0.1-SNAPSHOT.jar
+CLASS_PATH=com.intel.analytics.bigdl.ppml.e2e.examples.SimpleEncryptIO
+KMS_SERVER_PORT=3000
```

Comment:
does KMS_SERVER_PORT also need to be passed as arguments?

Reply:
Because the port number of KMS server is deployable by users, so now it is a tunable parameter that needed to be passed to SimpleEncryptIO.